templates/masterclasses.html

```diff
@@ -12,6 +12,13 @@ <h1 class="">Masterclasses</h1>
           Register your session
         </a>
       </p>
+      <form class="p-search-box">
+        <label class="u-off-screen" for="search">Search</label>
+        <input type="search" id="search" class="p-search-box__input" name="search" placeholder="Search" required=""
+          autocomplete="on">
+        <button type="reset" class="p-search-box__reset"><i class="p-icon--close">Close</i></button>
```

Comment:
The reset doesn't actually bring back the cards, it only empties the search input